client/src/main/java/com/influxdb/client/OrganizationsApi.java

```diff
@@ -119,6 +119,14 @@
     @Nonnull
     List<Organization> findOrganizations();
 
+    /**
+     * List all organizations.
+     *
+     * @return List all organizations
+     */
+    @Nonnull
+    List<Organization> findOrganizations(OrganizationsQuery query);
```

Comment:
Add `@Nonnull` annotation:

```suggestion
    List<Organization> findOrganizations(@Nonnull final OrganizationsQuery query);
```